### ISSUE TITLE
Spielfigur-Aktion: Sprinten

### ISSUE THREAD

**Issue by Smeagle** [Smeagle/dungeongame#35]
Die Spielfigur kann mit der Aktion zusätzliche Schritte laufen (2?).
Die Aktion löst in einem gewissen Radius Alarm aus (dass der Alarm nur mit einer bestimmten Wahrscheinlichkeit ausgelöst wird, kann man immer noch dazu machen...).

**Comment by murchandamus** [comment]
Im Brettspiel waren es AFAIR zufällig 0-3 Schritte. Weiß nicht mehr genau wie der Würfel verteilt war.

**Comment by Smeagle** [comment]
Ja, aber mein Vorschlag wäre, dass wir das erstmal anders machen weil ich es unlogisch finde, dass man würfelt wie weit man sprintet. Deshalb würde ich sagen dass man eine feste Anzahl Schritte sprinten kann (oder weniger). Und dass erstmal auf jeden Fall Alarm ausgelöst wird.

Sebastian hatte sich schon was mit dem Würfel gedacht. Aber ich bin mir nicht sicher, ob das wirklich nötig ist. Deshalb probieren wir es erstmal so und lösen die Probleme dann wenn wir sie vor Augen haben.

**Comment by murchandamus** [comment]
Okay, habe ein paar Aktionen schon im #38. Alarm ist noch nicht drin, aber kann man dann mal machen.

**Comment by murchandamus** [comment]
Sprint wurde über #38 hinzugefügt. Es werden zwei Felder addiert.
